app/config/locale/ro.countries.php

```diff
@@ -0,0 +1,198 @@
+    'GR' => 'Grecia',
+    'GD' => 'Grenada',
+    'GT' => 'Guatemala',
+    'GY' => 'Guyana',
+    'HN' => 'Honduras',
+    'HR' => 'Croația',
+    'HT' => 'Haiti',
+    'HU' => 'Ungaria',
+    'ID' => 'Indonesia',
+    'IN' => 'India',
+    'IE' => 'Irlanda',
+    'IR' => 'Iran',
+    'IQ' => 'Iraq',
+    'IS' => 'Islanda',
+    'IL' => 'Israel',
+    'IT' => 'Italia',
+    'JM' => 'Jamaica',
+    'JO' => 'Jordan',
+    'JP' => 'Japonia',
+    'KZ' => 'Kazakhstan',
+    'KE' => 'Kenya',
+    'KG' => 'Kyrgyzstan',
+    'KH' => 'Cambodia',
+    'KI' => 'Kiribati',
+    'KN' => 'Saint Kitts and Nevis',
+    'KR' => 'Coreea de Sud',
+    'KW' => 'Kuwait',
+    'LA' => 'Laos',
+    'LB' => 'Lebanon',
+    'LR' => 'Liberia',
+    'LY' => 'Libya',
+    'LC' => 'Saint Lucia',
+    'LI' => 'Liechtenstein',
+    'LK' => 'Sri Lanka',
+    'LS' => 'Lesotho',
+    'LT' => 'Lituania',
+    'LU' => 'Luxembourg',
+    'LV' => 'Letonia',
+    'MA' => 'Morocco',
+    'MC' => 'Monaco',
+    'MD' => 'Moldova',
+    'MG' => 'Madagascar',
+    'MV' => 'Maldive',
+    'MX' => 'Mexico',
+    'MH' => 'Insulele Marshall',
+    'MK' => 'Macedonia',
+    'ML' => 'Mali',
+    'MT' => 'Malta',
+    'MM' => 'Myanmar',
+    'ME' => 'Muntenegro',
+    'MN' => 'Mongolia',
+    'MZ' => 'Mozambique',
+    'MR' => 'Mauritania',
+    'MU' => 'Mauritius',
+    'MW' => 'Malawi',
+    'MY' => 'Malaysia',
+    'NA' => 'Namibia',
+    'NE' => 'Niger',
+    'NG' => 'Nigeria',
+    'NI' => 'Nicaragua',
+    'NL' => 'Olanda',
+    'NO' => 'Norvegia',
+    'NP' => 'Nepal',
+    'NR' => 'Nauru',
+    'NZ' => 'Noua Zeelandă',
+    'OM' => 'Oman',
+    'PK' => 'Pakistan',
+    'PA' => 'Panama',
+    'PE' => 'Peru',
+    'PH' => 'Philippines',
+    'PW' => 'Palau',
+    'PG' => 'Papua Noua Guinea',
+    'PL' => 'Polonia',
+    'KP' => 'Coreea de Nord',
+    'PT' => 'Portugalia',
+    'PY' => 'Paraguay',
+    'QA' => 'Qatar',
+    'RO' => 'Romania',
+    'RU' => 'Rusia',
+    'RW' => 'Rwanda',
+    'SA' => 'Arabia Saudită',
+    'SD' => 'Sudan',
+    'SN' => 'Senegal',
+    'SG' => 'Singapore',
+    'SB' => 'Insulele Solomon',
+    'SL' => 'Sierra Leone',
+    'SV' => 'El Salvador',
+    'SM' => 'San Marino',
+    'SO' => 'Somalia',
+    'RS' => 'Serbia',
+    'SS' => 'Sudanul de Sud',
+    'ST' => 'São Tomé și Príncipe',
+    'SR' => 'Suriname',
+    'SK' => 'Slovakia',
+    'SI' => 'Slovenia',
+    'SE' => 'Suedia',
+    'SZ' => 'Swaziland',
+    'SC' => 'Seychelles',
+    'SY' => 'Syria',
+    'TD' => 'Chad',
+    'TG' => 'Togo',
+    'TH' => 'Tailanda',
+    'TJ' => 'Tajikistan',
+    'TM' => 'Turkmenistan',
+    'TL' => 'Timor-Leste',
+    'TO' => 'Tonga',
+    'TT' => 'Trinidad și Tobago',
+    'TN' => 'Tunisia',
+    'TR' => 'Turcia',
+    'TV' => 'Tuvalu',
+    'TZ' => 'Tanzania',
+    'UG' => 'Uganda',
+    'UA' => 'Ucraina',
+    'UY' => 'Uruguay',
+    'US' => 'Statele Unite ale Americii',
+    'UZ' => 'Uzbekistan',
+    'VA' => 'Vatican',
+    'VC' => 'Saint Vincent și Grenadines',
```

Comment:
Another alternative is "Sfântul Vincențiu și Grenadinele" following the wikipedia article (https://ro.wikipedia.org/wiki/Sf%C3%A2ntul_Vincen%C8%9Biu_%C8%99i_Grenadinele), but both are correct :)